tmt/base.py

```diff
@@ -1371,6 +1389,11 @@ def show(self) -> None:
                 continue
             if value not in [None, [], {}]:
                 echo(tmt.utils.format(key, value))
+
+            # Show test instructions for manual tests in verbose mode
+            if key == "manual" and self.manual and self.verbosity_level:
```

Comment:
Any idea why `tmt -v tests show` has `self.verbosity_level == 1` but `tmt tests show -v` has it `self.verbosity_level == 0`?

Reply:
I got really puzzled that I have to change verbosity of whole `tmt` as `show -v` doesn't show any instructions